.github/workflows/codeql-analysis.yml

```diff
@@ -2,12 +2,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, 'b[0-9].[0-9]+' ]
```

Comment:
It's only "ironic" in the Alanis Morissett sense, but there's something about having to backport to each branch a change to make the tools work on each branch that "has that feeling". 😆

Reply:
Yeah...the fact that we have a configuration on each branch which current says, "don't run this on this branch," has that feeling, too -- why not just omit the configuration from the branch??

I know the answer to that question, and it's the same reason why this branch is getting a template instead of a custom version of this file....